package.json

```diff
@@ -20,6 +20,7 @@
     "@tsconfig/node18": "^1.0.1",
     "@types/jest": "^29.4.0",
     "@types/node": "^18.13.0",
+    "@types/serviceworker": "^0.0.67",
```

Comment:
We don't seem to actually use this package.

Reply:
This one is necessary for Typescript to understand the `ServiceWorkerGlobalScope` and `WorkerNavigator` types.

Reply:
where do we use those types? I must have overlooked them. Mind linking me?

Reply:
nm i found one